test/s.jl

```diff
@@ -25,5 +25,129 @@ using LinearAlgebra: I
+                    23.420823371828,
+                    -175.81072152220804,
+                    23.420819946054046,
+                ]
+            ]
+            for (expected, method) in zip(expecteds, methods)
+                spoint, trgepc, srfvec = subslr(method, "Mars", et, "IAU_MARS", "Earth", abcorr="LT+S")
+                spglon, spglat, spgalt = recpgr("mars", spoint, re, f)
+
+                @test spgalt ≈ expected[1]
+                @test rad2deg(spglon) ≈ expected[2]
+                @test rad2deg(spglat) ≈ expected[3]
+                spcrad, spclon, spclat = reclat(spoint)
+                @test rad2deg(spclon) ≈ expected[4]
+                @test rad2deg(spclat) ≈ expected[5]
+                sunpos, sunlt = spkpos("sun", trgepc, "iau_mars", "mars", abcorr="LT+S")
+                supgln, supglt, supgal = recpgr("mars", sunpos, re, f)
+                @test rad2deg(supgln) ≈ expected[6]
+                @test rad2deg(supglt) ≈ expected[7]
+                supcrd, supcln, supclt = reclat(sunpos)
+                @test rad2deg(supcln) ≈ expected[8]
+                @test rad2deg(supclt) ≈ expected[9]
+            end
+        finally
+            kclear()
+        end
+    end
+
+    @testset "subpnt" begin
+        try
+            furnsh(
+                path(CORE, :lsk),
+                path(CORE, :pck),
+                path(CORE, :spk),
+            )
+            et = str2et("2008 aug 11 00:00:00")
+            re, _, rp = bodvrd("MARS", "RADII", 3)
+            f = (re - rp) / re
+            methods = ["Intercept:  ellipsoid", "Near point: ellipsoid"]
+            expecteds = [
+                [
+                    349199089.604657, 
+                    349199089.64135259, 
+                    0.0, 
+                    199.30230503198658, 
+                    199.30230503198658,
+                    26.262401237213588, 
+                    25.99493675077423, 
+                    160.69769496801342, 
+                    160.69769496801342,
+                    25.994934171245205, 
+                    25.994934171245202,
+                ],
+                [
+                    349199089.6046486, 
+                    349199089.60464859, 
+                    0.0, 
+                    199.30230503240247, 
+                    199.30230503240247,
+                    25.99493675092049, 
+                    25.99493675092049, 
+                    160.69769496759753, 
+                    160.69769496759753,
+                    25.729407227461937, 
+                    25.994934171391463,
+                ]
+            ]
+            for (expected, method) in zip(expecteds, methods)
+                spoint, trgepc, srfvec = subpnt(method, "Mars", et, "IAU_MARS", "earth", abcorr="LT+S")
+                odist = norm(srfvec)
+                @test odist ≈ expected[2]
+                spglon, spglat, spgalt = recpgr("mars", spoint, re, f)
+                @test spgalt ≈ expected[3]
```

Comment:
When testing whether a value is close to zero you should explicitly set the absolute tolerance to something like `atol=sqrt(eps())` (see e.g. [here](https://www.python.org/dev/peps/pep-0485/#absolute-tolerance-default).

```suggestion
                @test spgalt ≈ expected[3] atol=sqrt(eps())
```

Otherwise the tests will fail on some platforms https://travis-ci.org/JuliaAstro/SPICE.jl/jobs/489938605